pkg/util/check/assert/http_response.go

```diff
@@ -10,50 +10,57 @@ import (
 )
 
 func ResponseMatchesFile(file string, successMsg, failureMsg string, otherFiles ...string) curl.HTTPResponseCheckFunc {
-	return func(t test.TestHelper, resp *http.Response, responseBody []byte, duration time.Duration) {
+	return func(t test.TestHelper, resp *http.Response, responseBody []byte, responseErr error, duration time.Duration) {
```

Comment:
You are changing an util method of http response. Have you updated all existing test cases which use this method ?